Products/Archetypes/skins/archetypes/edit_macros.pt

```diff
@@ -80,11 +80,12 @@
             method="post"
             enctype="multipart/form-data"
             class="enableUnloadProtection enableAutoFocus atBaseEditForm"
+            data-pat-autotoc="levels: legend; section: fieldset; className: autotabs"
             action=""
             id=""
             tal:attributes="action python:context.absolute_url()+'/'+template.id;
                             id string:${portal_type}-base-edit;
-                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
+                            class python:test(path('allow_tabbing|nothing'), 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection pat-autotoc', 'enableUnloadProtection enableAutoFocus enableUnlockProtection');">
```

Comment:
I know you tried to only modify what you needed but isn't it a good chance to modernize this idiom? Like, `python: 'enableUnloadProtection enableAutoFocus enableFormTabbing enableUnlockProtection pat-autotoc' if allow_tabbing else 'enableUnloadProtection enableAutoFocus enableUnlockProtection'`